styles.css

```diff
@@ -8,6 +8,7 @@ body{
   font-family
 }
 header, footer, aside, .main{
+  margin: 0;
```

Comment:
close, this question is asking for only the direct children in these elements, can you think of a CSS selector that would help target direct children?